src/pages/library/datasetBuilder/DatasetBuilderHeader.ts

```diff
@@ -0,0 +1,30 @@
+import { div, h, h1 } from 'react-hyperscript-helpers';
+import { Link } from 'src/components/common';
+import colors from 'src/libs/colors';
+import * as Nav from 'src/libs/nav';
+import { PAGE_PADDING_HEIGHT, PAGE_PADDING_WIDTH } from 'src/pages/library/datasetBuilder/constants';
+
+export const DatasetBuilderHeader = ({ name }: { name: string }) => {
```

Comment:
Because the load is done one component up, we want to pass the information the header needs. The actual dataset object lives in higher up. (And we can change this to be a `Dataset` object if we want down the road.)